composer/utils/checkpoint.py

```diff
@@ -416,7 +421,11 @@ def read_data(self, plan: LoadPlan, planner: LoadPlanner):
                 # Call function to modify state_dict
                 ignore_keys(model_state_dict)
 
-            dist_cp.load_state_dict(model_state_dict, storage_reader)
+            dist_cp.load_state_dict(
+                state_dict=model_state_dict,
+                storage_reader=storage_reader,
+                planner=load_planner,
+            )
```

Comment:
Just to confirm, this works on all versions of torch >=2? No gating, right? Test implies this but want to be sure

Reply:
Yes